docker/validator-dynamic/build.sh

```diff
@@ -5,5 +5,5 @@ set -e
 
 DIR="$( cd "$( dirname "$0" )" && pwd )"
 
-$DIR/../libra-build.sh $DIR/../validator/Dockerfile libra_e2e "$@"
-$DIR/../libra-build.sh $DIR/Dockerfile libra_validator_dynamic
+$DIR/../libra-build.sh $DIR/../validator/Dockerfile libra/validator "$@"
+$DIR/../libra-build.sh $DIR/Dockerfile libra/validator_dynamic
```

Comment:
and deleting this.

Reply:
this one might be tricky without handle a PR for the operational component that is out of scope of this repo :/